AdminWebsite/AdminWebsite/ClientApp/src/app/booking/search-email/search-email.component.ts

```diff
@@ -58,13 +59,19 @@ export class SearchEmailComponent implements OnInit, OnDestroy {
     }
 
     async getEmailPattern() {
-        const settings = await this.configService.getClientSettings().toPromise();
-        this.invalidPattern = settings.test_username_stem;
-        if (!this.invalidPattern || this.invalidPattern.length === 0) {
-            this.logger.error(`${this.loggerPrefix} Pattern to validate email is not set`, new Error('Email validation error'));
-        } else {
-            this.logger.info(`${this.loggerPrefix} Pattern to validate email is set with length ${this.invalidPattern.length}`);
-        }
+        this.$subscriptions.push(
```

Comment:
As above, no need for subscriptions observable 👍

Reply:
Done